api_fhir_r4/converters/coverageConverter.py

```diff
@@ -180,38 +143,37 @@ def __map_status(cls, code):
             2: R4CoverageConfig.get_status_active_code(),
             4: R4CoverageConfig.get_status_suspended_code(),
             8: R4CoverageConfig.get_status_expired_code(),
+            None: R4CoverageConfig.get_status_idle_code(),
```

Comment:
Please check the profile, The statuses are:
  * active: policy active or expired
  * cancelled: policy suspended
  * draft: policy idle

Reply:
ok. @dragos-dobre  I will check and update the code.